app/Http/Controllers/LessonController.php

```diff
@@ -75,7 +76,7 @@ public function adminDoCreate(Request $request): Application|RedirectResponse|Re
             'age_max' => 'required|integer|gte:age_min',
             'season_start' => 'required|date',
             'season_end' => 'required|date|after_or_equal:season_start',
-            'price' => 'required|numeric|min:0',
+            'pricing_structure'=>'required',
```

Comment:
Should probably check that the price structure exists in DB

Reply:
Seems like unnecessary work? It is a select input generated based on the values in the db, so you can't input anything that doesn't exist